src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
Do you mean you are giving up on this part of the pull request (i..e you intend to address the other comments), or the entire pull request?